### PR TITLE
[ISSUE #4086] Anonymous new can be replaced with lambda.[AsyncPubClient]

### PR DESCRIPTION
**Pull request by HattoriHenzo**
Fixes https://github.com/apache/eventmesh/issues/4086

### Motivation

Anonymous new can be replaced with lambda.[AsyncPubClient]

### Documentation

Refactoring of this part of the code

### CODE REVIEW

File: eventmesh-connectors/eventmesh-openfunction-connector/src/main/java/org/apache/eventmesh/connector/openfunction/client/CallbackServiceGrpc.java [hunk 1]
```diff
@@ -24,16 +41,16 @@ private CallbackServiceGrpc() {
 
     // Static method descriptors that strictly reflect the proto.
     @io.grpc.ExperimentalApi("https://github.com/grpc/grpc-java/issues/1901")
-    public static final io.grpc.MethodDescriptor<org.apache.eventmesh.common.protocol.grpc.cloudevents.CloudEvent,
+    public static final io.grpc.MethodDescriptor<CloudEvent,
```
Comment (Alonexc) [inline eventmesh-connectors/eventmesh-openfunction-connector/src/main/java/org/apache/eventmesh/connector/openfunction/client/CallbackServiceGrpc.java:44]: There are also substitutions in this class that can be made in this way.[line45,89,90,101,102,133,144...]

**Comment by xwm1992** [comment]
@HattoriHenzo hello, please fix the conflicts and leave your contact information for easy communication， such as email.

**Comment by HattoriHenzo** [comment]
> @HattoriHenzo hello, please fix the conflicts and leave your contact information for easy communication, such as email.

@xwm1992 Sorry for the delay. I have made a fix. Here is my email: reachmeongithub@yahoo.com

**Comment by xwm1992** [comment]
> > @HattoriHenzo hello, please fix the conflicts and leave your contact information for easy communication, such as email.
> 
> @xwm1992 Sorry for the delay. I have made a fix. Here is my email: [reachmeongithub@yahoo.com](mailto:reachmeongithub@yahoo.com)

@HattoriHenzo hi, I have sent an email to this address, please take a look, thanks.

**Review by xwm1992 — APPROVED** [review]

**Review by Alonexc — APPROVED** [review]

**Review by lrhkobe — APPROVED** [review]
LGTM

**Review by mxsm — APPROVED** [review]

**Pull request merged**